public/index.html

```diff
@@ -39,5 +39,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script src="https://cdn.cloudmersive.com/jsclient/cloudmersive-image-client.js"></script>
```

Comment:
is there a more react-y way of doing this? besides going into `index.html` and adding the script tag?

Reply:
Yes, usually you would import this as a module (e.g. from npm or yarn) and add `import moduleName from 'module-package-name'` at the top of the react component file where you are use it. 

The only package I can find is this one: https://www.npmjs.com/package/cloudmersive-image-api-client. The instructions are for Node but you can try:
- `npm install cloudmersive-image-api-client --save`
- add this to the top of  `src/common/components/ImageUploader/ImageUploader.js`: `import cloudmersiveImageApiClient from 'cloudmersive-image-api-client'

and see if that works!

Reply:
Womp womp, I'm running into a module error when using the package, which can be solved by configuring webpack a lil' bit (https://github.com/swagger-api/swagger-codegen/issues/3336). But I definitely don't feel comfortable doing that at this point 😅 .